packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+
+---
+
+## Product Explorer
+
+The Product Explorer is designed to be a catalog of Twilio products each with a short description,
+a link to the first page in the product in Console, and an external link to docs for that product
+(twilio.com/docs/&lt;product&gt;). This page provides a short description of what a product does, shows if the
+product is available in more than the US1 region, and allows you to pin them to the left navigation.
+
+The Product Explorer shows one card per product, regardless of how many different offerings exist in your
+product. There is a one-to-one relationship between the product card and the product that gets pinned to
+the left navigation.
+
+When introducing a new product to the Console, work closely with the Console team to determine the
+appropriate category to include your product in (testing may be required).
+
+Product description should be between 35-60 characters to fit (see checklist below).
+
+Products that appear in the explorer page should be products that you develop with and relate to your
+Twilio application. If you have questions about whether your product should be included here,
+reach out to the Console team.
+
+![](../../../assets/images/experiences/console/product-explorer.png)
+
+<ImageCaption>A screenshot of the Product Explorer page showing a catalog of Twilio products by category</ImageCaption>
+
+---
+
+## Search ("Jump to...")
+
+The "jump to" bar in the Console masthead area allows users to quickly jump to a specific page in the Console.
+The results will automatically populate with your published product pages. It is recommended that you also add
+keywords to help with findability of your product pages.
+
+- The JSON data driving search is located here: https://code.hq.twilio.com/twilio/console-navigation-backend/blob/main/json/consoleSearch.json
+- Update the search JSON and create a PR in the repository
+
+## Copy
+
+All Console copy should be presented in sentence case. For in-depth copy guidelines,
+see [Paste's Content Guidelines](/foundations/content/content-checklist).
+
+### Naming
+
+Partner with the Console team for support whenever you are creating navigation and menu names in Console to
+ensure they are maintaining a consistent experience and are meaningful to users.
+
+Naming should ideally be 22 characters or less to fit on a single line and cannot exceed 44 characters (2 lines).
+
+The H1 title of a page should correlate with the name in the sidebar navigation, although you can use a
+shortened version of the name in the sidebar navigation to save space, when necessary. Navigation items
+should be succinct and descriptive. Avoid using commonly shared terms like “Programmable” in the navigation
+as it inhibits scannability. It is ok, for example, to say “Messaging” in the navigation but
+“Programmable Messaging” in the H1 of the page.
+
+## Console product pre-launch checklist
```

Comment:
I know this would deviate from the doc, but what would you think of switching the "pre-launch checklist" and "Regional" sections? Feels more natural for the pre-launch checklist to be last.

Reply:
I agree, I switched them. I did think having Regional be the ending was a little anticlimactic somehow.